mirage/serializers/file.ts

```diff
@@ -19,15 +21,28 @@ export default class FileSerializer extends ApplicationSerializer<File> {
             },
         };
 
-        if (model.target && model.target.id && model.kind === 'folder') {
+        if (model.targetId && model.targetId.id && model.kind === 'folder') {
```

Comment:
Can move this kind==='folder' check inside of this first if-statement.
```if (model.targetId && model.targetId.id) {```

and move the 
```
if (model.kind === 'folder') { 
   returnValue.files = {
   ```

Reply:
Note: We check to see if the target exists here because quickfiles are associated to users, and have no target